### PR TITLE
feat:add-inv-functions

### PR DESCRIPTION
**Pull request by andybeltz**
### Summary

- Added `Remove-AriaNetworksLdapConfiguration` cmdlet to remove a LDAP configuration in VMware Aria Operations for Networks.
- Added `Test-AriaNetworksInternalAuthentication` cmdlet to test authentication with the internal API for a VMware Aria Operations for Networks instance.

### Type

- [ ] Bugfix
- [X] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [X] Documentation has been added or updated.

### CODE REVIEW

File: docs/documentation/functions/aria-suite/aria-operations-networks/Remove-AriaNetworksLdapConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Syntax
+
+```powershell
+Remove-AriaNetworksLdapConfiguration
+```
+
+## Description
+
+The Remove-AriaNetworksLdapConfiguration cmdlet allows a user to remove the LDAP configuration from VMware Aria Operations for Networks.
```
Comment (GaryJBlake) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Remove-AriaNetworksLdapConfiguration.md:15]:
```suggestion
The `Remove-AriaNetworksLdapConfiguration` cmdlet allows a user to remove the LDAP configuration from VMware Aria Operations for Networks.

```

File: docs/documentation/functions/aria-suite/aria-operations-networks/Test-AriaNetworksInternalAuthentication.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+## Syntax
+
+```powershell
+Test-AriaNetworksInternalAuthentication [-server] <String> [-user] <String> [-pass] <String>
+ [<CommonParameters>]
+```
+
+## Description
+
+The Test-AriaNetworksInternalAuthentication cmdlet checks authentication to a VMware Aria Operations for Networks
```
Comment (GaryJBlake) [inline docs/documentation/functions/aria-suite/aria-operations-networks/Test-AriaNetworksInternalAuthentication.md:16]:
```suggestion
The `Test-AriaNetworksInternalAuthentication` cmdlet checks authentication to a VMware Aria Operations for Networks

```

File: mkdocs.yml
Comment (GaryJBlake) [inline mkdocs.yml:1]: Not seeing Test-AriaNetworksInternalAuthentication added to the file?

**Review by GaryJBlake — CHANGES_REQUESTED** [review]

File: docs/documentation/functions/aria-suite/aria-operations-networks/Test-AriaNetworksInternalAuthentication.md (hunk 2 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Test-AriaNetworksInternalAuthentication.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/42261f7bbfc80fe8925418f1c844be4895a4d95f

File: mkdocs.yml
Comment (andybeltz) [reply mkdocs.yml:1]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/42261f7bbfc80fe8925418f1c844be4895a4d95f

File: docs/documentation/functions/aria-suite/aria-operations-networks/Remove-AriaNetworksLdapConfiguration.md (hunk 1 above)
Comment (andybeltz) [reply docs/documentation/functions/aria-suite/aria-operations-networks/Remove-AriaNetworksLdapConfiguration.md]: Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/42261f7bbfc80fe8925418f1c844be4895a4d95f

**Review by tenthirtyam — APPROVED** [review]
Updates LGTM! I've also rebased with `develop`.

**Review by GaryJBlake — APPROVED** [review]
LGTM!

**Pull request merged**